assembly/src/parsers/context.rs

```diff
@@ -467,6 +467,7 @@ impl ParserContext {
             "sdepth" => simple_instruction(op, Sdepth),
             "locaddr" => io_ops::parse_locaddr(op),
             "caller" => simple_instruction(op, Caller), // TODO: error if not in SYSCALL (issue #551)
+            "clk" => io_ops::parse_clk(op),
```

Comment:
It looks like we should be able to use `simple_instruction` here instead of writing a `parse` method.

Reply:
done